deploy/crds/workspaces.ecd.eclipse.org_devworkspacetemplates_crd.yaml

```diff
@@ -0,0 +1,491 @@
+          properties:
+            commands:
+              description: Predefined, ready-to-use, workspace-related commands
+              items:
+                properties:
+                  composite:
+                    description: Composite command
+                    properties:
+                      alias:
+                        type: string
+                      attributes:
+                        additionalProperties:
+                          type: string
+                        type: object
+                      commands:
+                        description: The commands that comprise this composite command
+                        items:
+                          type: string
+                        type: array
+                      label:
+                        type: string
+                      parallel:
+                        type: boolean
+                    type: object
+                  custom:
+                    description: Custom command
+                    properties:
+                      alias:
+                        type: string
+                      attributes:
+                        additionalProperties:
+                          type: string
+                        type: object
+                      commandClass:
+                        type: string
+                      embeddedResource:
+                        type: object
+                      label:
+                        type: string
+                    required:
+                    - commandClass
+                    - embeddedResource
+                    type: object
+                  exec:
+                    description: Exec command
+                    properties:
+                      alias:
+                        type: string
+                      attributes:
+                        additionalProperties:
+                          type: string
+                        type: object
+                      commandLine:
+                        description: The actual command-line string
+                        type: string
+                      component:
+                        description: Describes component to which given action relates
+                        type: string
+                      label:
+                        type: string
+                      workdir:
+                        description: Working directory where the command should be
+                          executed
+                        type: string
+                    required:
+                    - commandLine
+                    type: object
+                  type:
+                    description: Type of workspace command
+                    enum:
+                    - Exec
+                    - VscodeTask
+                    - VscodeLaunch
+                    - Custom
+                    type: string
+                  vscodeLaunch:
+                    description: VscodeLaunch command
```

Comment:
`VS Code launch command`?

Reply:
I should obviously have added a real description here :-)
But in fact this is generated from the existing description that was already there in the `DevWorkspaceTemplateSpec` Api Go struct. The only new thing is that this struct is now used in 2 main Custom Resource APIs.
 
So this change is not really part of the source changes done for this PR.

Do you think we could postpone additional fixes for missing / bad descriptions in the overall GO API to a distinct issue ?